gnucash/import-export/aqb/dialog-ab-trans.c

```diff
@@ -388,6 +403,45 @@ gnc_ab_trans_dialog_new(GtkWidget *parent, GNC_AB_ACCOUNT_SPEC *ab_acc,
                            _("Originator BIC (Bank Code)"));
         break;
 
+#if (AQBANKING_VERSION_INT >= 60400)
+    case SEPA_INTERNAL_TRANSFER:
+        gtk_label_set_text(GTK_LABEL (heading_label),
+                           /* Translators: Strings from this file are
+                              needed only in countries that have one of
+                              aqbanking's Online Banking techniques
+                              available. This is 'OFX DirectConnect'
+                              (U.S. and others), 'HBCI' (Germany),
+                              or 'YellowNet' (Switzerland). If none of
+                              these techniques are available in your
+                              country, you may safely ignore strings
+                              from the import-export/hbci
+                              subdirectory. */
+                           _("Enter a SEPA Internal Transfer"));
+        gtk_label_set_text(GTK_LABEL(recp_account_heading),
+                           _("Recipient IBAN (International Account Number)"));
+        gtk_label_set_text(GTK_LABEL(recp_bankcode_heading),
+                           _("Recipient BIC (Bank Code)"));
+
+        gtk_label_set_text(GTK_LABEL(orig_account_heading),
+                           _("Originator IBAN (International Account Number)"));
+        gtk_label_set_text(GTK_LABEL(orig_bankcode_heading),
+                           _("Originator BIC (Bank Code)"));
+	/* Disable target account entry for SEPA internal transfers, but only let choose from templates */
+    	gtk_widget_set_sensitive(td->recp_name_entry, FALSE);
+    	gtk_widget_set_sensitive(td->recp_account_entry, FALSE);
+    	gtk_widget_set_sensitive(td->recp_bankcode_entry, FALSE);
+    	gtk_widget_set_sensitive(add_templ_button, FALSE);
+    	gtk_widget_set_visible(add_templ_button, FALSE);
+    	gtk_widget_set_can_focus(add_templ_button, FALSE);
+    	gtk_widget_set_sensitive(del_templ_button, FALSE);
+    	gtk_widget_set_visible(del_templ_button, FALSE);
+    	gtk_widget_set_can_focus(del_templ_button, FALSE);
+        gtk_label_set_text(GTK_LABEL(template_label),
+                           _("Target Accounts"));
```

Comment:
Sorry, but that is the same kind of alignment used in the other cases in this switch for gtk_label_set_text()

Reply:
That's weird. Yesterday line 429 was indented one space less that the other statements.

Reply:
Yes, some of the whitspace and indentation problems you mentioned weren't definitely not in my local code, I have a hunch that either the interactive rebase function in Eclipse I used for squashing my git history messed that up, or there was something else. Must be, e.g. one of the indentation problems was because normal spaces suddenly were tab stops. Anyway I think those style problems should all be fixed now.

Reply:
It needs much time to disable the "replace n spaces by tabs" function in all Eclipse modules globally and project specific. ;-)